src/wwwroot/partials/signup/confirmationPage.html

```diff
@@ -18,24 +18,24 @@
       <div class="flex margin--bottom-inputs">
         <div class="wrapper-input width--full margin--right-inputs">
           <label class="label--regular" for="industry">{{'industry_text' | translate}}:</label>
-          <ui-select ng-model="selected.value" name="industry" title="{{'select_choose_industry' | translate}}" on-select="form.industry.$modelValue = $model" required>
+          <ui-select id="confirmation-select-industry"  ng-model="selected.value" name="industry" title="{{'select_choose_industry' | translate}}" on-select="form.industry.$modelValue = $model" required>
```

Comment:
@mvillaseco @andresmoschini as discussed with some UI developers, we believe that is a good practice to add identifiers to the form fields. They also facilitate the tests development.
In order to avoid collisions, the Id is created as [controllerName]-[inputType]-[fieldName]